Amazon.QLDB.Driver/result/BaseIonEnumerator.cs

```diff
@@ -98,7 +98,7 @@ public IIonValue Current
         internal TimingInformation? GetTimingInformation()
         {
             return this.processingTimeMilliseconds == null ? null :
-                new TimingInformation(this.processingTimeMilliseconds.Value);
```

Comment:
could we change them to similar things we did in `internal IOUsage? GetConsumedIOs()`?

Reply:
Sure, done in a68ed0b.